src/material/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
```

Comment:
Looks like this has been included by accident.

Reply:
I figured it's worth ignoring since it's just a comment, presumably coming from a new `yarn` or somehting

Reply:
I'm unsure how this lock file would have been generated. Though, I agree that it's okay ignoring since this PR only targets the `9.0.x` branch.